### PR TITLE
Fix issue with updates to columnDefinitions

### PR DESCRIPTION
**Pull request by scottwalstead**
Fix issue with updates to columnDefinitions such as reset to default and column reordering

### Summary
There is a bug in 7.5.1 where column reordering and reset to default in Table config doesn't work because of the changes I made to columnDefinitions being in the component state and not updating with nextProps
This fixes that bug and updates the column definitions

### Related Issues
Closes: https://github.com/procore/particles/issues/418

### Testing
Try to reorder columns and click reset to default with version 7.5.2 and see if it works

### CODE REVIEW

**Comment by scottwalstead** [comment]
Will rebase and reversion branch

File: table/src/Table.jsx [hunk 1]
```diff
@@ -107,11 +107,13 @@ export default class extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
+    console.log(nextProps);
```
Comment (ben-burlingham) [inline table/src/Table.jsx]: oops

File: table/README.md [hunk 2]
```diff
@@ -4,6 +4,9 @@ A table component with self-managed checkboxes, sticky head, sticky right panel,
 
 # Change Log
 
+##### 7.5.2
```
Comment (ben-burlingham) [inline table/README.md]: `.2` already published

File: table/src/Table.jsx (hunk 1 above)
Comment (scottwalstead) [reply table/src/Table.jsx]: removed it

**Pull request merged**